dotnet-desktop-guide/net/winforms/controls/how-to-add-to-a-form.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
Is there a better name for this file, e.g. how-to-add-controls.md?

Reply:
@gewarren  AFAIK we're not supposed to repeat words in the full URI so `controls` would be there twice.